### PR TITLE
create a standardtracer package (and some minor fallout)

### PR DESCRIPTION
**Pull request by bensigelman**
See #4 and the corresponding points that @yurishkuro made in #2.

### CODE REVIEW

File: examples/dapperish/dapper.go [hunk 1]
```diff
@@ -9,8 +9,15 @@ import (
 	"sync"
 
 	"github.com/opentracing/api-golang/opentracing"
+	"github.com/opentracing/api-golang/opentracing/standardtracer"
 )
 
+func NewDapperishTracer(processName string) opentracing.OpenTracer {
```
Comment (bensigelman) [inline examples/dapperish/dapper.go]: (per suggestion!)

**Comment by bensigelman** [comment]
I am going to :zzz: now, but I'm open to criticism about package names, etc :)

I do think that this is a good general direction, though... nice to keep the `opentracing` package proper as thin as possible.

File: examples/dapperish/dapper.go (hunk 1 above)
Comment (yurishkuro) [inline examples/dapperish/dapper.go]: I think golint might complain about stutter on this one. You probably want just this: `dapperish.NewTracer()`

Comment (bensigelman) [inline examples/dapperish/dapper.go]: Ok, sure.

**Comment by yurishkuro** [comment]
lgtm

**Pull request merged**